src/index.html

```diff
@@ -53,8 +54,20 @@
               },
               onComplete: function(event) {
                 // callback for when everything is complete
-              }
-            })
+              },
+              steps: [
```

Comment:
More pressingly, I'm not sure this file should be here at all. Definitely not in its current state.